src/PropertyDetail.js

```diff
@@ -1,33 +1,40 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {useParams} from "react-router-dom";
-import lowConditionParcelsData from './data/lowPoorWornOutConditionParcels.json'
-import outOfCityParcelsData from "./data/ownerInNebraskaOutOfOmahaParcels.json";
-import outOfNebraskaParcelsData from "./data/ownerOutOfNebraskaParcels.json";
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
 import Typography from '@material-ui/core/Typography';
 import Link from '@material-ui/core/Link';
 import Box from '@material-ui/core/Box';
 import Breadcrumbs from '@material-ui/core/Breadcrumbs';
+import ReactGA from "react-ga";
+import lowConditionParcelsData from './data/lowPoorWornOutConditionParcels.json'
+import outOfCityParcelsData from "./data/ownerInNebraskaOutOfOmahaParcels.json";
+import outOfNebraskaParcelsData from "./data/ownerOutOfNebraskaParcels.json";
 
 export default function PropertyDetail() {
 
-  const {source, pin} = useParams();
-  let selectedProperty = {};
-  let propertyCount = 0;
-  if (source === "out-of-state") {
-    selectedProperty = outOfNebraskaParcelsData.find(property => property.PIN === pin);
-    propertyCount = outOfNebraskaParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  } else if (source === "out-of-omaha") {
-    selectedProperty = outOfCityParcelsData.find(property => property.PIN === pin);
-    propertyCount = outOfCityParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  } else if (source === "low-condition") {
-    selectedProperty = lowConditionParcelsData.find(property => property.PIN === pin);
-    propertyCount = lowConditionParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  }
+  const [selectedProperty, setSelectedProperty] = useState({QUALITY: "", CONDITION: ""});
+  const [propertyCount, setPropertyCount] = useState(0);
+  const [encodedPropertyAddress, setEncodedPropertyAddress] = useState("");
+  const [violationLinks, setViolationLinks] = useState([]);
 
-  const encodedPropertyAddress = encodeURIComponent(`${selectedProperty.ADDRESS_LA}, ${selectedProperty.PROP_CITY}, NE ${selectedProperty.PROP_ZIP}`);
-  const violationLinks = selectedProperty.VIOLATION_LINKS || [];
+  const {source, pin} = useParams();
+  useEffect(() => {
+    ReactGA.initialize('UA-175185008-1');
```

Comment:
That sweet sweet Google Analytics :+1: .  What google account is this associated with?

Reply:
lol, i just copy-pasta'd from App.js... I assume somebody knew :)

Reply:
This is associated with my gmail account. Happy to switch it up, if we have better suggestions.